packages/eds-core-react/src/components/Popover/PopoverHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { Divider } from '../Divider'
+
+export type PopoverHeaderProps = HTMLAttributes<HTMLDivElement>
+
+const StyledPopoverHeader = styled.div`
+  display: grid;
```

Comment:
What if we use `display:flex` and `justify-content: space-between`? 🤔

Reply:
Yes works nice 😊